apps/cyberstorm-nextjs/.env.development

```diff
@@ -2,3 +2,8 @@
 # the security implications that entails.
 NEXT_PUBLIC_API_DOMAIN=http://thunderstore.localhost
 NEXT_PUBLIC_API_URL=$NEXT_PUBLIC_API_DOMAIN/api/cyberstorm/
+CYBERSTORM_AUTH_ENABLED=
+CYBERSTORM_AUTH_SECRET=development
+CYBERSTORM_AUTH_DISCORD_URL=https://auth.thunderstore.io/auth/login/discord/?next=https%3A%2F%2Fbeta.thunderstore.io%2F
```

Comment:
Should these URL variable names have the `NEXT_PUBLIC` prefix?

(Also I think having separate variable for each provider is 100% fine here since there's only three and we're not planning to add more but generally speaking the repetition could be avoided by having one variable with the URL up to `.../login/` and another with the query parameter.)

Reply:
1. I left the `NEXT_` prefix out of it, so that it wouldn't be confused with `next-auth` stuff 🤔 I'd say the `CYBERSTORM_AUTH_` prefix is fitting here 🤷 

2. Fixd

Reply:
1. I'm referring to this: https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser . (If we'd be talking about just good naming without the aforementioned functional perspective, I'd argue that prefixing everything with Cyberstorm is a bit pointless since the whole app is cyberstorm at this point, and misleading since cyberstorm initially referred only to the component library.)
2. You didn't fix the query parameter part.

Reply:
1. Oh, neat, didn't know about that. Yes those should.
2. Fixd, for real now. Also made it so that the query param is added separately, since we probably at some point want to change it dynamically 👍